api/server.go

```diff
@@ -21,6 +21,8 @@ func NewServer(store *db.Store) *Server {
 
 	subrouter.POST("/users", handler.CreateUsersHandler(store))
 	subrouter.GET("/users/:id", handler.GetUsersHandler(store))
+	subrouter.GET("/users", handler.ListUsersHandler(store))
+	subrouter.POST("/users/", handler.UpdateUsersHandler(store))
```

Comment:
I believe that the HTTP method `POST` is intended to create new resources 🧐
Multiple identical `POST` requests result in different request to the server.

`PUT` is intended to update a resource at a specific URL.
Multiple identical `PUT` requests have the same effect as a single request to the server.

It's strongly recommend to follow best practices!

Reply:
I made the changment and updated the curl request example.